examples/decam-13k.ini

```diff
@@ -0,0 +1,379 @@
+; DECADE SGC, and DES Y3. We do not use shear ratios
+; when analyzing DES Y3.
+[runtime]
+sampler   = polychord
+verbosity = standard
+
+; This parameter is used several times in this file, so is
+; put in the DEFAULT section and is referenced below as %(2PT_FILE)s
+[DEFAULT]   
+2PT_FILE_DES   = ${COSMOSIS_DIR}/likelihood/des-y3/2pt_NG_final_2ptunblind_02_26_21_wnz_maglim_covupdate.fits
+2PT_FILE_NGC   = ${COSMOSIS_DIR}/likelihood/decade/2pt_decade_ngc_20241128.fits
+2PT_FILE_SGC   = ${COSMOSIS_DIR}/likelihood/decade/2pt_decade_sgc_20250218.fits
+
+[pipeline]
+modules =  consistency bbn_consistency
+           camb fast_pt
+           load_nz_fits_des         load_nz_fits_ngc         load_nz_fits_sgc
+           source_photoz_bias_des   source_photoz_bias_ngc   source_photoz_bias_sgc
+           IA_des                   IA_ngc                   IA_sgc
+           pk_to_cl_des             pk_to_cl_ngc             pk_to_cl_sgc
+           add_intrinsic_des        add_intrinsic_ngc        add_intrinsic_sgc 
+           2pt_shear_des            2pt_shear_ngc            2pt_shear_sgc
+           shear_m_bias_des         shear_m_bias_ngc         shear_m_bias_sgc
+           2pt_des_like             2pt_ngc_like             2pt_sgc_like
+          
+likelihoods = 2pt_des 2pt_ngc 2pt_sgc
+timing=F
+debug=F
+priors = ${COSMOSIS_DIR}/examples/decam-13k-priors.ini
+values = ${COSMOSIS_DIR}/examples/decam-13k-values.ini
+extra_output = cosmological_parameters/sigma_8 cosmological_parameters/sigma_12 
+               data_vector/2pt_des_chi2 data_vector/2pt_ngc_chi2 data_vector/2pt_sgc_chi2
+               likelihoods/2pt_des_like likelihoods/2pt_ngc_like likelihoods/2pt_sgc_like 
+               data_vector/desi_bao_chi2 data_vector/desy5sn_chi2 likelihoods/desi_bao_like likelihoods/desy5sn_like
+
+
+; It's worth switching this to T when sampling using multinest, polychord,
+; or other samplers that can take advantage of differences in calculation speeds between
+; different parameters.
+fast_slow = F
+first_fast_module = 
+
+[polychord]
+resume = F
+feedback = 3
+fast_fraction = 0.1
+base_dir = ${OUTDIR}/checkpoints
+polychord_outfile_root = decam-13k
+;Settings for paper runs
+live_points = 500
+num_repeats=100
+tolerance=0.01
+
+[output]
+filename = ${OUTDIR}/decam-13k.txt
+format   = text
+
+[consistency]
+file = ${COSMOSIS_DIR}/utility/consistency/consistency_interface.py
+
+[camb]
+file = ${COSMOSIS_DIR}/boltzmann/camb/camb_interface.py
+mode = power
+lmax = 2500          ;max ell to use for cmb calculation
+feedback=0         ;amount of output to print
+AccuracyBoost=1.1 ;CAMB accuracy boost parameter
+do_tensors = T
+do_lensing = T
+NonLinear = pk
+halofit_version = mead2020
+zmin_background = 0.
+zmax_background = 5.
+nz_background = 401
+kmin=1e-4
+kmax = 50.0
+kmax_extrapolate = 500.0
+nk=700
+
+[bbn_consistency]
+file = ${COSMOSIS_DIR}/utility/bbn_consistency/bbn_consistency.py
+
+[load_nz_fits_des]
+file = ${REPO_DIR}/modules/load_nz_fits.py
```

Comment:
I think these REPO_DIRs can now be replaced with COSMOSIS_DIR?  Actually I think you can remove all of them too (and the leading slash) and assume we are running from the CSL directory.